pombola/kenya/views.py

```diff
@@ -139,6 +139,19 @@ def get_context_data(self, **kwargs):
             if shujaaz.FINALISTS_DICT[year].get(self.object.pk)
         ]
 
+        political_positions = self.object.position_set.all().political().currently_active()
+
+        na_memberships = political_positions.filter(title__slug='member-national-assembly')
+        senate_memberships = political_positions.filter(title__slug='senator')
+        if na_memberships:
```

Comment:
I just wonder if it'd be worth throwing an exception if there's > 1 `na_memberships` or > 1 `senate_memberships` so we'll know there's something to fix?